File: athrill-account-client/projects/at-models/src/lib/users/user-role.model.ts
import { UserRoleLinkModel } from './user-role-link.model';


export class UserRoleModel{
    
    public UserRoleId : number;
    public NameKey : string;
    public DisplayName: string; 
    public Description : string;
    public IsShown : boolean;
        
    public UserRoleLinks : UserRoleLinkModel[];
}

File: athrill-account-server/Libraries/Model/Users/UserRoleModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AT.Model.Users
{
    public class UserRoleModel
    {
        public int UserRoleId { get; set; }
        public string NameKey { get; set; }
        public string DisplayName { get; set; }
        public string Description { get; set; }
        public bool IsShown { get; set; }

        #region Links
        public  ICollection<UserRoleLinkModel> UserRoleLinks { get; set; }
        #endregion

        #region override
        public int Id => UserRoleId;
        public override string ToString()
        {
            return DisplayName;
        }
        #endregion
    }
}
